js/details.js

```diff
@@ -0,0 +1,43 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const countryDetailsSection = document.querySelector(".country-details");
+    const loader = document.querySelector(".loader");
+  
+    // Extract country name
+    const urlParams = new URLSearchParams(window.location.search);
+    const countryName = urlParams.get("country");
+  
```

Comment:
You dont need to add this comment, you already have a variable names `countryName`